package.json

```diff
@@ -8,7 +8,8 @@
   },
   "scripts": {
     "dev-start": "webpack --config webpack.config.js --watch --watch-poll",
-    "dev-server": "nodemon server/index.js"
+    "dev-server": "nodemon server/index.js",
+    "seed": "node server/dbSeed.js"
```

Comment:
I can see below that the path to the dbSeed.js is server/database/dbSeet.js. However, here I see server/dbSeed.js. 

Did you make sure that this command works?

Reply:
Great find! I'll amend this to the correct path.

Reply:
@miraclestyle Thanks for finding this. I've amended and confirmed it works!